### DISCUSSION TITLE
New version of Outlook

### DISCUSSION THREAD

**Discussion by rayakajamon** [Set-OutlookSignatures/Set-OutlookSignatures#105] [Q&A] [answered]
Hi, will this solution work with the new Outlook for windows?

https://support.microsoft.com/en-us/office/getting-started-with-the-new-outlook-for-windows-656bb8d9-5a60-49b2-a98b-ba7822bc7627

Thank you

**Comment by GruberMarkus** [comment, marked as answer]
Yes, Set-OutlookSignatures supports Outlook, Outlook Web and New Outlook.

Certain features may require the Benefactor Circle add-on, Set-OutlookSignatures will inform you about this requirement when trying to use such a feature.

Please let me know if this answers your question.

**Reply by rayakajamon** [reply]
Thank you so much.  Yes, it does answer my question.  Really appreciate you, sir!
